### PR TITLE
Fix/edit profile

### PR DESCRIPTION
**Pull request by mneokvsenorm**

### CODE REVIEW

File: src/components/ProfileView/index.js [hunk 1]
```diff
@@ -31,13 +41,19 @@ const ProfileView = ({ user, classes, toggle }) => (
...
     </Card>
   </div>
 );
 
 ProfileView.propTypes = {
-  user: PropTypes.object.isRequired,
-  toggle: PropTypes.func.isRequired,
+  user: userType.isRequired,
+  toggle: func.isRequired,
 };
```
Comment (AndrZakhara) [inline src/components/ProfileView/index.js]: You must describe all props in prop-types.

File: src/components/UploadImage/index.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+        const { onImageUploadSuccess, onImageUploadFailed } = this.props;
+
+        if (!file) {
+          return;
+        }
+
+        const fileUploadTask = distRef.child(file.name).put(file);
+
+        fileUploadTask
+          .then(async () => {
```
Comment (AndrZakhara) [inline src/components/UploadImage/index.js]: Remove API query from a component.

Comment (mneokvsenorm) [reply src/components/UploadImage/index.js]: Ok, tnx

File: src/components/ProfileView/index.js (hunk 1 above)
Comment (mneokvsenorm) [reply src/components/ProfileView/index.js]: Ok

**Pull request merged**